content/en/docs/contribute/review/reviewing-prs.md

```diff
@@ -124,6 +124,12 @@ When reviewing, use the following as a starting point.
 ### Language and grammar
 
 - Are there any obvious errors in language or grammar? Is there a better way to phrase something?
+  - Focus on the language and grammar of the parts of the page that the author is changing.
+     Unless the author is clearly aiming to update the entire page, they have no obligation to
+     fix every issue on the page.
+  - If you are reviewing a pull request and you spot grammar, spelling, or other style errors that
+    are not related to the PR's purpose, consider opening an issue about fixing those unrelated issues.
```

Comment:
The purpose of any documentation PR is to communicate information clearly. This includes using correct grammar, spelling, and style. This guideline effectively puts language review outside the scope of most PRs.

It is seldom worth the extra time and effort to create separate issues the types of issues that language reviews uncover. It is better to resolve issues in the present ("fix as you go" philosophy) by _fixing_ or _triaging_ it. This is what triaging looks like: 
- If the issues are nits, then fixing them requires little effort. 
- If the author disagrees with or doesn't understand the nit/suggestion, then it's probably not worth the time/effort to discuss the issue, and the author should triage it.

Reply:
My understanding of what @sftim updates meant is that, as a reviewer, you should focus on reviewing the parts of the page that are being updated. That content should be reviewed for technical and editorial correctness. If you find errors on the page that dont directly relate to what the PR author is attempting to address, then it should be treated as a separate issue.

IMHO its is not always reasonable to ask a PR author to update ALL errors on a page, technical or editorial. If the author agrees to work on an issue that impacts a part of a page, they have agreed to that scope of work. As a reviewer, if you expand the scope of the review to the whole page, then you are asking the PR author to take on more work than they agreed to and may be adding barriers to their contribution getting accepted. 

If the PR author is attempting to make updates to the whole page, by doing an editorial update, or fixing typos, then i think its reasonable to point out errors that they maybe didn't find throughout the page(s) being updated.

From my perspective, we always have folks looking for good first issues to work on and this can help more good first issues get created. By having a diverse set of issues, we are encouraging contributors of all levels to get involved in the community.

Reply:
> &hellip;as a reviewer, you should focus on reviewing the parts of the page that are being updated. That content should be reviewed for technical and editorial correctness. If you find errors on the page that dont directly relate to what the PR author is attempting to address, then it should be treated as a separate issue.

Yep, that's the point I want to make.
I think there's a similar point about this in [Docs For Developers](https://docsfordevelopers.com/) (co-authored by our our emeritus contributors @jaredbhatti and @zacharysarah).

Reply:
@a-mccarthy I'm going to borrow some of your wording here. Thanks.

Reply:
I see - My concern was based on the interpretation that "not related to the PR's purpose" is ambiguous and can be arbitrarily determined. I've seen PR authors reject language reviews of the lines they modified content because they considered the reviewer's suggestions outside the purpose of their changes. I think we should provide a more concrete definition, such as: "Limit your suggestions to only the added or modified content. Avoid making suggestions in the unmodified portions of a file. If you see problems in the unmodified content, consider opening a new issue to address them."

Reply:
How do you like the revised language @rolfedh?

Reply:
>  I think we should provide a more concrete definition, such as: "Limit your suggestions to only the added or modified lines. Avoid making suggestions in the unmodified lines in a file. If you see problems in the unmodified lines, consider opening a new issue to address them."

(nit) Just noting that GitHub won't let reviewers add a suggestion to sections of the file that have not been modified
(nit) To counter my previous point 😉  , with trivial edits, this PR links to a page (I agree to link to the [page](https://www.kubernetes.dev/docs/guide/pull-requests/#trivial-edits))that states:

> If you find one grammatical or spelling error, it is likely there are more in that file, you can really make your Pull Request count by checking the formatting, checking for broken links, and fixing errors and then submitting all the fixes at once to that file.
> 
> Some questions to consider:
> 
>     Can the file be improved further?
>     Does the trivial edit greatly improve the quality of the content?

In a way, we will be asking (and have asked) folks who make trivial edits to review other sections of the page

Imo, I welcome the changes proposed

Reply:
If someone proposes a trivial edit to a page and then realises what they've done, then usually the appropriate thing for them to do is to find all the other nit-level fixes that the page needs. Typically there's at least one more lurking somewhere.

This does actually align with our review advice: when a contributor's stated intent is to find **all** the nits, and they miss one, then a reviewer *should* point that out (the exception about only reviewing specific updates only applies when the intent of the PR is to change only part of the page).
What matters is _not_ which lines change. Instead, the reviewer should mainly consider the author intent as described in the PR title and its description.
I'm fine with someone clarifying this detail about why these two aims don't actually conflict.

Thanks to future Rey for adding the point about blog reviews.

Reply:
This sounds to me as if it's resolved enough to close this off.